docs/content/_docs/query_language.html

```diff
@@ -244,52 +202,175 @@ <h4>Operators</h4>
   </tr>
 
   <tr>
-    <th scope="row">Tag Starts With</th>
-    <td><pre><code class="language-hql">&lt;a&gt; ^ &lt;b&gt;</code></pre></td>
-    <td><pre><code class="language-json">["^", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+    <th scope="row">Key Matches</th>
+    <td><pre><code class="language-hql">$key = &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["key", &lt;a&gt;]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Key Does Not Match</th>
+    <td><pre><code class="language-hql">$key != &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["key", &lt;a&gt;]]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">[true]</code></pre></td>
+    <td><pre><code class="language-json">[true]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag False</th>
+    <td><pre><code class="language-hql">[false]</code></pre></td>
+    <td><pre><code class="language-json">[false]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">RegEx On Tag</th>
+    <td><pre><code class="language-hql">&lt;a&gt; ~ &lt;b&gt;</code></pre></td>
+    <td><pre><code class="language-json">["~", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">Raw</th>
+    <td>Custom filters</td>
+    <td><pre><code class="language-hql">q &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["q", &lt;a&gt;]</code></pre></td>
+  </tr>
+  
+</table>
+</div>
+
+<h4 id="customfilters">
+  Custom Filters
+  <a class="link-to" href="#customfilters"><span class="glyphicon glyphicon-link"></span></a>
+</h4>
+
+<p>
+  When you need to select series using a more complex syntax than that available through the basic tag suggestions, you can create custom filters.
+</p>
+
+<div class="table-responsive">
+<table class="table table-bordered">
+  <tr>
+    <th></th>
+    <th>Description</th>
+    <th>Parsed</th>
+    <th>JSON</th>
+  </tr>
+
+  <tr>
+    <th scope="row">Simple String</th>
+    <td>Simple string without quotes.</td>
```

Comment:
What's your thought on separating these from the Operators sections? Like what makes something an operator vs a custom filter? Some also seem to be duplicated between the two (like tag mataches).